### DISCUSSION TITLE
Open Shell with 22H2 (KB5019980) on Windows 11

### DISCUSSION THREAD

**Discussion by lawrenmr** [Open-Shell/Open-Shell-Menu#1206] [General]
I've been using Open Shell and ExplorerPatcher with Windows 11 for some time without issue.
However, when Cumulative Update for Windows 11 22H2 (KB5019980) - the one with the tabbed Explorer windows - came out, I installed it and explorer.exe kept crashing every few minutes...

_Faulting application name: Explorer.EXE, version: 10.0.22621.755, time stamp: 0xbcd85d13
Faulting module name: ClassicExplorer64.dll, version: 4.4.170.0, time stamp: 0x627d583b
Exception code: 0xc0000005
Fault offset: 0x000000000001af79
Faulting process ID: 0x0x28D0
Faulting application start time: 0x0x1D8E91E1DF32B6E
Faulting application path: C:\WINDOWS\Explorer.EXE
Faulting module path: C:\Program Files\Open-Shell\ClassicExplorer64.dll
Report ID: 24d347be-8471-4e23-94ed-8ee3db2ee001_

I uninstalled KB5019980 and all returned to normal.

Did a little digging and the only suggestion was to clear Explorer's cache > Options > General > Privacy > Clear
So, a couple of weeks later, I did this and re-installed KB5019980 and so far, so good, explorer.exe doesn't appear to crash anymore.

One feature I've found though is that clicking on Start Menu > Start Menu (Windows) which normally shows the Windows 11 traditional menu now does nothing, but I rarely use this anyway, so nothing lost there for me.

Hope this helps someone.

**Comment by blackcrack** [comment]
hmm.. try to uninstall openshell, uninstall explorerpatcher too, update then the system (there exist a new version fom ExplorerPatcher.. have become at last a mail for a new release..) then install the latest explorer patcher again and reinstall openshell.. ans say what's happens..

best regards
blacky

**Reply by lawrenmr** [reply]
Have uninstalled OpenShell and ExplorerPatcher, rebooted, added ExplorerPatcher, then OpenShell and the start menu is working again.
Let's just hope windows explorer stays up.

Thanks.

**Comment by obuolinis** [comment]
I lost Win+X menu after installing the same KB. What I did to fix it was re-run OpenShell installer and choose Repair.
Using OpenShell in combination with ExplorerPatcher.

**Comment by blackcrack** [comment]
https://github.com/valinet/ExplorerPatcher/releases/tag/22621.819.52.1_02cb6e9

>22621.819.52.1 Pre-release
>@[valinet](https://github.com/valinet) valinet released this 17 Nov 01:20
>22621.819.52.1_02cb6e9
>02cb6e9
>
>Tested on OS builds 22621.819 and 22000.1098.
>
>Please make sure you are connected to the Internet while installing, the application might need to perform one-time downloads for several resources in order to enable full functionality.
>Details
>1
>
>  *  Windows 11 Start menu: Implemented centering on screen when taskbar is not at the bottom. (4212e35)
>  *  Windows 11 taskbar: Option to use the stock taskbar context menu. (451db3c)
>  *  Fixed a bug that could display the Start menu on a wrong monitor or outside the screen when the taskbar was moved to the top of the screen and the previous setting was at the right edge of the screen. (53fad19)
>
>Please consult the README for more details.
>A detailed change log is available here.
>An archive containing all the files generated during the build process (including dxgi.dll and symbol files) is available here.
>
>This release has been published automatically from commits up to and including https://github.com/valinet/ExplorerPatcher/commit/02cb6e900cd75cdc4d702d23f882944804f41f11 in branch [master](https://github.com/valinet/ExplorerPatcher/tree/master/).
>

https://github.com/valinet/ExplorerPatcher/releases/

thank's 10 Vali !



feedbacks be ever pretty well in my opinion..
best

**Comment by Truchin** [comment]
For me ExplorerPatcher fixes well the bug of the toolbar which no longer moves from the left edge of the explorer.
The downside is that it's an extra application and it's not easy to configure, especially if you don't speak English.
Strongly an update of Open-Shell.
Thanks
